apps/api/src/indexing/blockchain-types.ts

```diff
@@ -0,0 +1,33 @@
+// Raw log from blockchain
+export type RawLog = {
+  address: `0x${string}`;
```

Comment:
if we _want_, `viem` exports a `Hex` type. (idk if that makes the type checker less friendly, though.)